lua/terrortown/lang/zh_hans.lua

```diff
@@ -1458,7 +1454,7 @@ L.button_delete_all_spawns = "删除所有生成点位置"
 L.label_dynamic_spawns_enable = "为该地图启用动态生成"
 L.label_dynamic_spawns_global_enable = "为所有地图启用自定义生成"
 
-L.header_equipment_weapon_spawn_setup = "武器生成设定"
+L.header_equipment_weapon_spawn_setup = "武器生成设置"
```

Comment:
there where multiple translations for this key and your tool let the first one win, although ingame the last won probably and this would change the current behaviour.

IMO this should be changed back.

Reply:
I can do that, yes. I could also change the tool to iterate backwards over the file